gbpservice/neutron/tests/unit/nfp/configurator/agents/test_generic_config.py

```diff
@@ -192,51 +183,56 @@ def _test_handle_event(self, ev):
-                driver, 'configure_routes') as mock_config_src_routes, \
+                driver, const.EVENT_CONFIGURE_ROUTES.lower(),
+                return_value=common_const.SUCCESS)) as (
+                                                    mock_config_src_routes), (
             mock.patch.object(
-                driver, 'clear_routes') as mock_delete_src_routes, \
+                driver, const.EVENT_CLEAR_ROUTES.lower(),
+                return_value=common_const.SUCCESS)) as (
+                                                mock_delete_src_routes), (
             mock.patch.object(
-                sc, 'poll_event') as mock_hm_poll_event, \
+                sc, 'poll_event')) as mock_hm_poll_event, (
             mock.patch.object(
-                driver, 'configure_healthmonitor', return_value='SUCCESS'), \
+                driver, const.EVENT_CONFIGURE_HEALTHMONITOR.lower(),
+                return_value=common_const.SUCCESS)), (
             mock.patch.object(
-                agent, '_get_driver', return_value=driver):
+                agent, '_get_driver', return_value=driver)):
 
-            if 'CONFIGURE_HEALTHMONITOR' in ev.id:
+            if const.EVENT_CONFIGURE_HEALTHMONITOR in ev.id:
                 ev.id, periodicity = ev.id.split()
 
             agent.handle_event(ev)
 
             resource_data = self.fo._fake_resource_data()
-            if ev.id == 'CONFIGURE_INTERFACES':
+            if ev.id == const.EVENT_CONFIGURE_INTERFACES:
                 mock_config_inte.assert_called_with(
                                         self.fo.context, resource_data)
-            elif ev.id == 'CLEAR_INTERFACES':
+            elif ev.id == const.EVENT_CLEAR_INTERFACES:
                 mock_clear_inte.assert_called_with(
                                         self.fo.context, resource_data)
-            elif ev.id == 'CONFIGURE_ROUTES':
+            elif ev.id == const.EVENT_CONFIGURE_ROUTES:
                 mock_config_src_routes.assert_called_with(
                             self.fo.context, resource_data)
-            elif ev.id == 'CLEAR_ROUTES':
+            elif ev.id == const.EVENT_CLEAR_ROUTES:
                 mock_delete_src_routes.assert_called_with(
                             self.fo.context, resource_data)
-            elif 'CONFIGURE_HEALTHMONITOR' in ev.id:
-                if periodicity == gen_cfg_const.INITIAL_HM_RETRIES:
+            elif const.EVENT_CONFIGURE_HEALTHMONITOR in ev.id:
+                if periodicity == const.INITIAL_HM_RETRIES:
                     mock_hm_poll_event.assert_called_with(
-                                ev, max_times=gen_cfg_const.INITIAL_HM_RETRIES)
-                elif periodicity == gen_cfg_const.FOREVER:
+                                ev, max_times=const.INITIAL_HM_RETRIES)
+                elif periodicity == const.FOREVER:
                     mock_hm_poll_event.assert_called_with(ev)
-            elif ev.id == 'CLEAR_HEALTHMONITOR':
```

Comment:
This was removed as it was just a placeholder. Anyways, I have written a test case for this.